lib/src/audio_file_waveforms.dart

```diff
@@ -62,6 +62,9 @@ class AudioFileWaveforms extends StatefulWidget {
   /// Allow seeking with gestures when turned on.
   final bool enableSeekGesture;
 
+  /// Allow seeking with gestures on pause mode.
+  final bool enableSeekGestureOnPauseMode;
```

Comment:
we do not need this variable

Reply:
so it means we didn't provide access to enable and disable this functionality?